### ISSUE TITLE
About the implement

### ISSUE THREAD

**Issue by lhao0301** [sciencefans/Beyond-Frontal-Faces#2]
I fine-tune the global body model on alexnet and it gets 36.8%(accuracy). I have seen that you got accuracy of 69% in your blog. Are there any improvement or tricks that you do to improve accuracy from 34% to 69%?

**Comment by rocksat** [comment]
同问，global body model 我用 pretrain alexnet 得到 37.25%, fine-tune 以后得到的准确率是50.1%，怎么也到不了69%，求问你如何做到的？有什么tricks吗？

**Comment by lhao0301** [comment]
@rocksat Code to split test set in `test.m` is not very reasonable. Try changing the division scheme. I got the consistent result after doing that.
